libs/utils/android/workloads/uibench.py

```diff
@@ -85,7 +85,7 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         self.out_dir = out_dir
         self.collect = collect
 
-        # Press Back button to be sure we run the video from the start
+        Screen.unlock(self._target)
         System.menu(self._target)
```

Comment:
About second comment, I will submit the other workload changes in other patch series please, so that this can be merged first. thanks